src/Module/Help.php

```diff
@@ -21,35 +21,35 @@ public static function content()
 		$text = '';
 		$filename = '';
 
-		$app = self::getApp();
-		$config = $app->getConfig();
+		$a = self::getApp();
```

Comment:
Don't you always add an comment like "we don't have to save string length" ;-)
=> but it's ok for me, just mentioned :D

Reply:
No, there was actually an issue in this method where some `$a` instances hadn't been converted to `$app`, so this wasn't a beauty correction. I could go either way (changing `$a` to `$app`) but the most prevalent spelling is `$a` throughout the codebase, hence this decision.

Reply:
OK